src/scenic/simulators/carla/model.scenic

```diff
@@ -71,10 +76,12 @@ class CarlaActor(DrivingObject):
         self.carlaActor.set_location(utils.scenicToCarlaLocation(pos, elevation))
 
     def setVelocity(self, vel):
-        self.carlaActor.set_velocity(utils.scenicToCarlaVector3D(*vel))
+        self.carlaActor.set_target_velocity(utils.scenicToCarlaVector3D(*vel))
```

Comment:
It does apply an instantaneous velocity to the actor, but it will never be exactly that value, as it is set before the forces that affect the actor are calculated, which end up reducing its final value by a small amount.